x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/openai/embeddings/OpenAiEmbeddingsRequestTaskSettings.java

```diff
@@ -40,12 +47,23 @@ public static OpenAiEmbeddingsRequestTaskSettings fromMap(Map<String, Object> ma
         ValidationException validationException = new ValidationException();
 
         String model = extractOptionalString(map, MODEL, ModelConfigurations.TASK_SETTINGS, validationException);
+        if (model != null) {
```

Comment:
We don't need to check a `logDeprecations` boolean here because this class is only used during a rest request, so we should always log.